test/common/ws/serverProcess.js

```diff
@@ -16,9 +16,9 @@
 var randomstring = require('randomstring');
 var SocketCluster = require('socketcluster');
 var MasterWAMPServer = require('wamp-socket-cluster/MasterWAMPServer');
-var sinon = require('sinon');
 var WSClient = require('./client');
 var Promise = require('bluebird');
+var sinon = require('sinon');
```

Comment:
This shouldn't be necessary now.

Reply:
The tests started to fail somehow even after making it global, so I added the sinon back and things started to work, weird behaviour.